src/Orckestra.Composer/Configuration/CacheConfigurationCategoryNames.cs

```diff
@@ -78,6 +78,8 @@ public static class CacheConfigurationCategoryNames
 
         public static string RecurringOrderPrograms = "RecurringOrderPrograms";
 
+        public static string CustomProfiles = "CustomProfiles";
```

Comment:
for new category add here the rule
src\Orckestra.Grocery.Website\App_Config\Caching.config
    <category name="CustomProfiles" profileName="LongLived" />